internal/net/call/call.go

```diff
@@ -385,25 +381,36 @@ func (rc *reconnectingConnection) Call(ctx context.Context, h MethodKey, arg []b
 }
 
 func (rc *reconnectingConnection) callOnce(ctx context.Context, h MethodKey, arg []byte, opts CallOptions) ([]byte, error) {
-	var hdr [msgHeaderSize]byte
-	copy(hdr[0:], h[:])
+	enc := codegen.NewEncoder()
```

Comment:
How about moving the header encoding/decoding code into routines of their own. The code here can then be:

hdr := encodeHeader(ctx, h)

Reply:
Done